tests/testdata/run/warn_on_deprecated_api/main.js

```diff
@@ -1,5 +1,6 @@
 import { runEcho as runEcho2 } from "http://localhost:4545/run/warn_on_deprecated_api/mod.ts";
 
+// @ts-ignore `Deno.run()` was soft-removed in Deno 2.
 const p = Deno.run({
```

Comment:
I think we should do something like this instead of `@ts-ignore` (ts-ignore isn't as precise and can sometimes ignore relevant diagnostics):

```
const p = (Deno as any).run({
```

Reply:
ts-ignore wasn't really created for purposes where you can use typescript itself to do a surgical ignore. I think we should actually introduce a `no-ts-ignore` lint rule because of how misused it is. I'll go throught he codebase later and fix all these.